release-notes.txt

```diff
@@ -0,0 +1,222 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-14
```

Comment:
```suggestion
Date: 2022-03-15
```

Reply:
since we start building tonight, the release will only be available for most platforms tomorrow morning

Reply:
will fix, release notes moved to a separate MR

Reply:
will fix, release notes moved to a separate MR